Examples/ObjectiveC/ExtrusionsExample.m

```diff
@@ -41,14 +40,14 @@ - (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
     layer.sourceLayerIdentifier = @"building";
     
     // Filter out buildings that should not extrude.
-    layer.predicate = [NSPredicate predicateWithFormat:@"extrude != false AND height >=0"];
```

Comment:
Awesome! I just edited it. Was running into issues because of https://github.com/mapbox/mapbox-gl-native/issues/9056